reana_server/rest/workflows.py

```diff
@@ -731,10 +678,31 @@ def upload_file(workflow_id_or_name):  # noqa
           description: >-
             Request succeeded. File successfully transferred.
           schema:
-            type: object
-            properties:
-              message:
-                type: string
+            type: file
```

Comment:
As mentioned in https://github.com/reanahub/reana-client/pull/134#discussion_r203789606 this returns a message with information [see](https://github.com/reanahub/reana-workflow-controller/blob/master/reana_workflow_controller/rest.py#L446).